app/views/users/_edit.haml

```diff
@@ -20,10 +20,11 @@
 
 
         = form_for "user", url: edit_user_path,
-          html: {method: :put, class: "form-horizontal col-md-12", id: "email-form"} do |f|
+          html: {method: :put, class: "form-horizontal", id: "email-form"} do |f|
           = f.error_messages
           .form-group
-            = f.text_field :email, value: @user.unconfirmed_email || @user.email, class: "col-md-7 form-control"
+            .col-sm-12
+              = f.text_field :email, value: @user.unconfirmed_email || @user.email, class: "form-control"
```

Comment:
Avoid using instance variables in partials views

Reply:
I'm going to ignore this, I didn't touch any variables and I don't want to change the whole file :shrug: